docs/.vuepress/helpers.js

```diff
@@ -19,6 +58,24 @@ function getVersions(buildMode) {
   return [...next, ...availableVersions];
 }
 
+/**
+ * Gets all available frameworks.
+ *
+ * @returns {string[]}
+ */
+function getFrameworks() {
+  return ['javascript', 'react', 'vue2', 'vue'];
```

Comment:
Is `angular` missing from the list on purpose?

Reply:
No, thanks. I'll change it.

Reply:
Changed within: https://github.com/handsontable/handsontable/pull/9371/commits/fe333de7510c3c7dd83b754be23caa6b2441e45a.